tfchain_bridge/pkg/substrate/events.go

```diff
@@ -24,3 +74,72 @@ func (s *SubstrateClient) SubscribeEvents() (*state.StorageSubscription, types.S
 	// defer unsubscribe(sub)
 	return sub, key, err
 }
+
+func (client *SubstrateClient) processEventsForHeight(height uint32) (Events, error) {
+	log.Info().Msgf("fetching events for blockheight %d", height)
+	records, err := client.GetEventsForBlock(height)
+	if err != nil {
+		log.Info().Msgf("failed to decode block with height %d", height)
```

Comment:
If you choose to log here already, it should be an Error log (or at least a Warn), as this is not expected behavior.